CHANGELOG.md

```diff
@@ -3,6 +3,101 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0-beta.2
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🏃 Add slot typings (#11795) @romgrk
+- 🎁 Support UTC date formatting in charts tooltip (#11943) @shaharyar-shamshi
```

Comment:
```suggestion
- 🎁 Support UTC date formatting in Charts tooltip (#11943) @shaharyar-shamshi
```

If we capitalize "Data Grid" and "Date and Time Pickers" I think we should also do it for "Charts"